client-app/src/examples/contact/detail/DetailsPanel.js

```diff
@@ -0,0 +1,116 @@
+import {div, filler, hbox, img, placeholder} from '@xh/hoist/cmp/layout';
+import {XH, hoistCmp, uses} from '@xh/hoist/core';
+import {DetailsPanelModel} from './DetailsPanelModel';
+import {Icon} from '@xh/hoist/icon/Icon';
+import './DetailsPanel.scss';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {form} from '@xh/hoist/cmp/form';
+import {formField} from '@xh/hoist/desktop/cmp/form';
+import {textArea, textInput} from '@xh/hoist/desktop/cmp/input';
+import {button} from '@xh/hoist/desktop/cmp/button/index';
+
+export const detailsPanel = hoistCmp.factory({
+    model: uses(DetailsPanelModel),
+    className: 'contact-details-panel',
+
+    render({model, className, ...props}) {
+        const {currentRecord} = model;
+
+        return panel({
+            title: currentRecord?.data.name ?? 'Select a contact',
+            className,
+            icon: Icon.detail(),
```

Comment:
This little misalignment is a bit unfortunate here - 

<img width="428" alt="Screen Shot 2021-05-19 at 4 32 46 PM" src="https://user-images.githubusercontent.com/225806/118897866-09a56d00-b8c0-11eb-9724-4f9a4a49814a.png">

I'm thinking we might not need / want the title at all - it doesn't add much and I think looks better (slightly) without. 

Alternative would be to force its height to line up with the toolbar - I am a bit sad that's not the case out of the box. Or to try with the detail panel below the grid in a more horizontal layout - we might want that if you add that tags column, so we have more width for the grid.